pymagicc/__init__.py

```diff
@@ -295,13 +289,13 @@ def run(scenario, output_dir=None,
         ``return_config`` is set to True
     """
 
-    model_run = MAGICC(output_dir)
+    magicc = MAGICC()
     try:
```

Comment:
if I'm not mistaken I think this should also work if you do something like

```
with MAGICC() as magicc:
    ...
    results = magicc.run()
```

and then you can drop the `finally` block as the `__exit__` method of `MAGICC` should take care of it

Reply:
Good point, pushed a change!